docs/getting-started.md

```diff
@@ -156,9 +171,9 @@ Now go to create Kintoblock page. Add a Kintoblock name _(name must be unique ac
 
 > For the rest of the guide we are going to assume the name is _`nodeexample`_
 
-> endpoints are going to be in the following format **http://api.staging.kintohub.com/{microservicename}**
+> endpoints are going to be in the following format **http://api.kintohub.com/{microservicename}**
```

Comment:
should this be a {KintoBlock} ? as we have on our prod examples. For consistency

Reply:
I would prefer keeping this as microservice, we should update everywhere else

more on it: before kintoblocks were microservices, but now they are bigger